src/components/customerCases/customerCases.module.css

```diff
@@ -9,8 +9,6 @@
   display: flex;
   flex-direction: column;
   gap: 5rem;
-  max-width: 1200px;
```

Comment:
If we set a max-width on this section it will become much more narrow than what's in the sketches. I am, however, using a wide screen here and we might want to set a max-width. I'll ask Julia or OP. 

with max width: 
<img width="1917" alt="Screenshot 2024-10-28 at 10 38 14" src="https://github.com/user-attachments/assets/f6cd7d9d-f70c-4c4c-99bc-2e30c7b891e9">

In sketches: 
<img width="928" alt="Screenshot 2024-10-28 at 10 39 17" src="https://github.com/user-attachments/assets/a8a46ddd-1ff4-4366-83fa-ad49a124aa32">

Reply:
Max-width will be 1400px